entity-framework/index.yml

```diff
@@ -41,11 +41,11 @@ highlightedContent:
     # Card
     - title: "API reference browser"
       itemType: reference
-      url: /dotnet/api/?view=efcore-6.0
+      url: /dotnet/api/?view=efcore-6.0&preserve-view=true
     # Card
     - title: "Explore other options for .NET Data"
       itemType: reference
-      url: dotnet-data
+      url: dotnet-data/index.yml
```

Comment:
This file needs exact file URL. Earlier it was the folder. Which in web world, tries to go to index.* and it worked by accident.